feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/proxy/RealMetaAccountsUpdatesRegistry.kt

```diff
@@ -57,6 +60,13 @@ class RealMetaAccountsUpdatesRegistry(
             .map { hasUpdates() }
     }
 
+    override fun observeConsumedUpdatesMetaIds(): Flow<Set<Long>> {
```

Comment:
`observeLastConsumedUpdatesMetaIds`
Otherwise it reads like we are accumulating all previosuly consumed ids